frontend/src/components/common/form-input/styles.module.scss

```diff
@@ -0,0 +1,8 @@
+.error {
+  font-size: 0.8rem;
```

Comment:
Use our default variables, for font-size and color. Also better to use bootstrap classes instead of custom styles.

And use just `styles.scss` not module, for easiest work with bootstrap